# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/Context_gl_4_4_autogen.h
# Commit: 3a91885d47e0
# Full Hash: 3a91885d47e0ef7fba9749562d52300eea67c651
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-11-11 21:52:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 77cee77c0307 -r 3a91885d47e0 gfx/angle/checkout/src/libANGLE/Context_gl_4_4_autogen.h
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/angle/checkout/src/libANGLE/Context_gl_4_4_autogen.h	Fri Nov 08 17:06:00 2019 +0000
@@ -0,0 +1,28 @@
+// GENERATED FILE - DO NOT EDIT.
+// Generated by generate_entry_points.py using data from gl.xml.
+//
+// Copyright 2019 The ANGLE Project Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+//
+// Context_gl_4_4_autogen.h: Creates a macro for interfaces in Context.
+
+#ifndef ANGLE_CONTEXT_GL_4_4_AUTOGEN_H_
+#define ANGLE_CONTEXT_GL_4_4_AUTOGEN_H_
+
+#define ANGLE_GL_4_4_CONTEXT_API                                                                   \
+    void bindBuffersBase(GLenum target, GLuint first, GLsizei count, const GLuint *buffers);       \
+    void bindBuffersRange(GLenum target, GLuint first, GLsizei count, const GLuint *buffers,       \
+                          const GLintptr *offsets, const GLsizeiptr *sizes);                       \
+    void bindImageTextures(GLuint first, GLsizei count, const GLuint *textures);                   \
+    void bindSamplers(GLuint first, GLsizei count, const GLuint *samplers);                        \
+    void bindTextures(GLuint first, GLsizei count, const GLuint *textures);                        \
+    void bindVertexBuffers(GLuint first, GLsizei count, const GLuint *buffers,                     \
+                           const GLintptr *offsets, const GLsizei *strides);                       \
+    void bufferStorage(GLenum target, GLsizeiptr size, const void *data, GLbitfield flags);        \
+    void clearTexImage(GLuint texture, GLint level, GLenum format, GLenum type, const void *data); \
+    void clearTexSubImage(GLuint texture, GLint level, GLint xoffset, GLint yoffset,               \
+                          GLint zoffset, GLsizei width, GLsizei height, GLsizei depth,             \
+                          GLenum format, GLenum type, const void *data);
+
+#endif  // ANGLE_CONTEXT_API_4_4_AUTOGEN_H_